### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7970,10 +7970,13 @@ instrumentaion->instrumentation
 instsallation->installation
 instuction->instruction
 instuments->instruments
+insturment->instrument
 instutionalized->institutionalized
 instutions->intuitions
+insue->ensue
 insufficent->insufficient
 insuffiency->insufficiency
+insure->ensure
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: e.g. health insurance

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9639,8 +9639,10 @@ instrumentaion->instrumentation
 instsallation->installation
 instuction->instruction
 instuments->instruments
+insturment->instrument
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the plural at least please.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -9639,8 +9639,10 @@ instrumentaion->instrumentation
...
 instutionalized->institutionalized
 instutions->intuitions
+insue->ensue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest insure.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -9664,6 +9666,8 @@ intelectual->intellectual
 intelegence->intelligence
 intelegent->intelligent
 intelegently->intelligently
+inteligable->intelligible
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also add intelligibility version.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -9664,6 +9666,8 @@ intelectual->intellectual
...
+inteligance->intelligence
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add intelligently version.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -9758,6 +9763,7 @@ interesst->interest
 interessted->interested
 interessting->interesting
 interfave->interface
+interfear->interfere
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add interference version.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: What?

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: I.e. interfearence->interference

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -9664,6 +9667,8 @@ intelectual->intellectual
 intelegence->intelligence
 intelegent->intelligent
 intelegently->intelligently
+inteligable->intelligible, intelligibility,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Sorry, I meant a separate suggestion of inteligability->intelligibility

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: I.e. inteligantly->intelligently

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -9758,6 +9764,7 @@ interesst->interest
 interessted->interested
 interessting->interesting
 interfave->interface
+interfear->interfere, interference,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: As a new suggestion sorry, i.e. interfearnce->interference and probably also interfearence->interference

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**